tracer/test/snapshots/AwsSnsTests.NetCore.SchemaV1.verified.txt

```diff
@@ -60,7 +60,7 @@
   {
     TraceId: Id_5,
     SpanId: Id_6,
-    Name: sns.request,
+    Name: aws.sns.send,
```

Comment:
Any chance you could add one example of an inbound SNS span and one example of an admin/request span?

Reply:
I was thinking of doing this, but I wondered if this should be addressed in another PR.

Since I'd have to create new methods to use reflection on in the integration.
Wdyt?

Reply:
I was thinking that in this PR it would be good to show the effects that the tags have on all 3 types of SNS spans we create, but can you elaborate more on the reflection you're talking about in the integration?

Reply:
I think you are right. That should happen indeed. But we currently don't have one method which this integration instruments.

Sorry if my wording was confusing. I was saying that I'd have to add more methods to instrument in this PR, although its main purpose is not doing that, that would be the only way to showcase the effects of the tags on the 3 types of SNS spans.

For reflection, I know that is the way on how things are autoinstrumented in .NET and Java.

Reply:
As per our 1:1, this will be handled in another PR when adding new methods to instrument.